architects/demo/website.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="website.default_website" model="website">
+        <field name="theme_id" ref="base.module_theme_real_estate"/>
+        <field name="name">Industry-architect</field>
```

Comment:
I think you need to add something like (or is by default?)
```xml
        <field name="logo" type="base64" file="architects/static/src/binary/website/1-logo"/>
        <field name="favicon" type="base64" file="architects/static/src/binary/website/1-favicon"/>
```

Reply:
The logo is not present in the original database